VRDR/DeathRecord.cs

```diff
@@ -492,7 +494,9 @@ private void CreateDeathLocation()
             DeathLocationLoc.Meta.Profile = deathlocation_profile;
             DeathLocationLoc.Type.Add(new CodeableConcept(CodeSystems.LocationType, "death", "death location", null));
             DeathLocationLoc.Name = DeathRecord.BlankPlaceholder; // We cannot have a blank string, but the field is required to be present
-            AddReferenceToComposition(DeathLocationLoc.Id, "DeathInvestigation");
+            if(Decedent!=null){
```

Comment:
Why is DeathLocationLoc being added to the Composition related to Decedent being present or not?

Reply:
@pkmitre: Only DeathCertificateDocument includes a Composition and a Decedent.  So, it looks like I'm using a Decedent as a proxy for this.   Perhaps, AddReferenceToComposition should just return if there is no Composition?